core/src/main/resources/application-dev.properties

```diff
@@ -0,0 +1,8 @@
+################################################################################
+# Configuration for development environment
+# Please set below properties!
+spring.datasource.url=jdbc:mysql://referral-1.cdaj903rkno2.us-west-1.rds.amazonaws.com/referral
+spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
+spring.datasource.username=referral
+spring.datasource.password=Referral&2020
```

Comment:
Should this be hardcoded and added here? Can we remove this?

Reply:
Yes, you are right, let me put an example password instead of real.